walk-in-interview/src/main/webapp/log-in/firebase-auth.js

```diff
@@ -65,6 +68,32 @@ function signIntoBusinessAccount(email, password) {
       });
 }
 
+/**
+ * This will add the firebase ui for phone authentication
+ * to the provided element.
+ *
+ * @param {String} elementId The div element to add the UI.
+ * @param {String} successPath The url for redirect on login success.
+ */
+function addPhoneAuthUI(elementId, successPath) {
```

Comment:
i think you should move this symbols into a namespace something like :
```
Auth  = {}
Auth.addPhoneAuthUI = function(...){...}
Auth.createApplicantAccount = ...
[...]
```

and export the Auth object instead of each individual methods. See how firebase did it:
```firebaseui.auth.AuthUI```